src/EditorFeatures/CSharpTest/PullMemberUp/CSharpPullMemberUpTests.cs

```diff
@@ -24,7 +24,26 @@ private async Task TestNoRefactoringProvidedAsync(string initialMarkUp)
 
```

Comment:
> "add test for crossing language scene"

scene?

Reply:
Should be 'scenario'

Reply:
ah thanks.  i was trying to figure it out, but that word didn't come to mind :)